website/docs/d/container_dedicated_host.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+
+subcategory: "Kubernetes Service"
+layout: "ibm"
+page_title: "IBM: container_dedicated_host"
+description: |-
+  Get information about a dedicated host.
+---
+
+# ibm_container_dedicated_host
+
+Retrieve information about a dedicated host. For more information about dedicated hosts, see [Creating and managing dedicated hosts on VPC Gen 2 infrastructure](https://cloud.ibm.com/docs/containers?topic=containers-dedicated-hosts).
+
+
+## Example usage
+In the following example, you can retrieve a dedicated host:
+
+```terraform
+data "ibm_container_dedicated_host" "test_dhost" {
+  host_id      = "<dedicated host id>"
```

Comment:
In example provider some values instead of "<dedicated host id>"
host_id      = "535625345466"